libyul/ObjectParser.cpp

```diff
@@ -121,18 +121,19 @@ optional<ObjectParser::SourceNameMap> ObjectParser::tryParseSourceNameMapping()
 
 	// Matches some "@use-src TEXT".
 	static std::regex const lineRE = std::regex(
-		"(^|\\s)@use-src\\s((.|[\r\n])*)",
+		"(^|\\s)@use-src\\s",
```

Comment:
It should still match if `@use-src` is exactly the documentation literal, so the `\\s` must be optional, so I think you need something like `@use-src(\\s|$)` (here, it is not important if `$` matches `\n` or the end of string, both work.